src/data/resolvers/status.ts

```diff
@@ -51,17 +56,35 @@ export const statusResolvers = ({
         return null;
       }
     },
-
     async isReputationOracleAlive() {
       try {
+        const latestRootHash = await networkClient.getReputationRootHash();
+        const metaColonyAddress = await networkClient.getMetaColony();
+        const colonyClient = await colonyManager.getClient(
+          ClientType.ColonyClient,
+          metaColonyAddress,
+        );
+        const { skillId } = await colonyClient.getDomain(ROOT_DOMAIN_ID);
         const res = await fetch(
-          `${reputationOracleEndpoint}/${networkNameForReputationOracle}`,
+          `${reputationOracleEndpoint}/${networkNameForReputationOracle}/${latestRootHash}/${metaColonyAddress}/${skillId}/${AddressZero}/noProof`,
         );
         return res.status === 200;
       } catch (error) {
         console.error(error);
         return null;
       }
     },
+    async isIPFSAlive() {
+      try {
+        if (!(isDev || process.env.DEV)) {
```

Comment:
This is just a stylistic preference but I would do this check in the component rather than here (I think that avoids some extra computation, I am not super sure though.)

Reply:
Whether we move this check to the component or we keep it here, we would need to do some extra computation, yes?  For example, if we moved this check to the component, due to the rule of hooks, we couldn't just do something like, unfortunately:

```
if (!(isDev || process.env.DEV)) {
    const { data: isReputationOracleAlive } = useReputationOracleLivenessQuery({
      pollInterval: networkCheckInterval,
    });
}
```

We would have to use something like a lazy query, and then conditionally request the data in a useEffect or something similar, so the difference in performance between these 2 methods may be marginal.